### PR TITLE
feat: adds millisecond-level timestamps to argo and workflow-controller

### PR DESCRIPTION
**Pull request by mikhailian**
Adding millisecond-level timestamps in follow-up to #2795 so that we have them everywhere and not just in argoexec. Thanks to @astein-te for implementing the original feature request.

### CODE REVIEW

File: cmd/workflow-controller/main.go [hunk 1]
```diff
@@ -108,6 +108,18 @@ func NewRootCommand() *cobra.Command {
 	return &command
 }
 
+func init() {
+	cobra.OnInitialize(initConfig)
+	fmt.Println("hello world")
```
Comment (simster7) [inline cmd/workflow-controller/main.go:113]: Remove this?

**Review by simster7 — CHANGES_REQUESTED** [review]
I'm wondering if we want to make this setting configurable, but I don't really see a point to doing so. Any thoughts?

File: cmd/workflow-controller/main.go (hunk 1 above)
Comment (mikhailian) [reply cmd/workflow-controller/main.go:113]: Thanks!

**Review by simster7 — APPROVED** [review]
Thank you!

**Pull request merged**